deflate_medium.c

```
@@ -73,16 +73,16 @@ static void insert_match(deflate_state *s, unsigned char *window, struct match m
 
         if (LIKELY(strstart >= match.orgstart)) {
             if (LIKELY(strstart + match_len - 1 >= match.orgstart)) {
-                insert_string(s, window, strstart, match_len);
+                insert_string_count(s, window, strstart, match_len);
             } else {
```

Cppcheck (2.21.0)

[style] The function 'zng_clz32' is never used.

(unusedFunction)